src/api/userAuth.ts

```
@@ -1,6 +1,8 @@
 'use client';
 import { signinUserInterface, signupUserInterface } from '@/api/userInterface';
+import { useStore } from '@/store/useAuthStore';
```

ESLint

[error] 'useStore' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)